tst/EnergyPlus/unit/Humidifiers.unit.cc

```diff
@@ -0,0 +1,144 @@
+// EnergyPlus::Humidifiers Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/FArray1D.hh>
+#include <ObjexxFCL/IOFlags.hh>
+#include <ObjexxFCL/gio.hh>
+#include <ObjexxFCL/bit.hh>
```

Comment:
@Nigusse I noticed bit.hh is included here as well, but I don't see where it is actually used in here.  Is it possible this (and the psychrometric change) is just an artifact from playing around?  Or is it intentional?  Thanks.